Italian locale

egueli opened the pull request:
Here are the library strings, translated in Italian.

I lack lots of context and I know little about AC technologies, so some translations may be off. In particular, the composed strings like `D_STR_EYEAUTO` make totally no sense to me (that one translates to `Occhio Auto`, literally `Auto Eye`).

It would be nice to have some context (i.e. where/how the original string is used), but I guess that's a common problem in every translation.

I tried to keep strings that look trademarked (like `6th Sense` or `Zone Follow`) as they are.

egueli committed:
Strings translated to Italian

crankyoldgit committed:
Fix linter issue

src/locale/it-IT.h:171:  Lines should be <= 80 characters long  [whitespace/line_length] [2]

crankyoldgit commented on src/locale/it-IT.h:
```diff
@@ -0,0 +1,176 @@
+#define D_STR_OFF "Spento"
+#define D_STR_MODE "Modalità"
+#define D_STR_TOGGLE "Alterna"
+#define D_STR_TURBO "Turbo"
```
FYI, where the word/spelling doesn't change, you don't need to include it.
I'll remove anything obvious that doesn't change.

crankyoldgit committed:
Remove unneeded words

Remove words that are the same as English (en-AU) and are not required due to use later.

crankyoldgit commented on src/locale/it-IT.h:
```diff
@@ -0,0 +1,157 @@
+#define D_STR_OFF "Spento"
+#define D_STR_MODE "Modalità"
+#define D_STR_TOGGLE "Alterna"
+#define D_STR_SLEEP "Stand-by"
```
This is the only one that doesn't look right.
"Sleep" here means, the act of sleeping. ie. What you do at night/go to bed. Not suspending a device to low power etc.

Google Translate suggests "Dormire". Correct?

egueli replied:
Are you sure about that? I saw some usages of `kSleepStr` and, by looking at the surrounding lines, they all seem related to some device mode, e.g. https://github.com/crankyoldgit/IRremoteESP8266/blob/bbf3738191982323da69f4433e89d84238b23caf/src/ir_Daikin.cpp#L2715, https://github.com/crankyoldgit/IRremoteESP8266/blob/f5a517464c5d870f811a7adc8bf6e0b2435b803a/src/ir_Neoclima.cpp#L426 or https://github.com/crankyoldgit/IRremoteESP8266/blob/bbf3738191982323da69f4433e89d84238b23caf/src/ir_Goodweather.cpp#L353

crankyoldgit replied:
Yes. It is often a special device mode. But it is a mode for when you are in bed sleeping.
e.g.
The mode on a number of devices will after a period, start to slowly increase the temperature (if in cooling mode) so it becomes more energy efficient etc. It does this till it reaches a certain level and either stays that way, or in some cases, turns off.

Hence, while it could be seen as a "a mode that will turn off" i.e. if or how long before it run for until it turns off. They often do more than just that. They are commonly used/exist for a mode while _you_ are sleeping, not so much the device.

egueli replied:
OK, I translated with "Sonno".

crankyoldgit replied:
Thanks

crankyoldgit reviewed (changes requested):
I've removed what I think are the redundant entries. Everything except `D_STR_SLEEP` look good to me (like I can tell anyway ;-)

Fix that one and I think we are good to merge.

egueli committed:
fix typo & reword

egueli committed:
Fix D_STR_SLEEP

crankyoldgit reviewed (approved):
LGTM

crankyoldgit commented:
The changes mentioned above have been included in the latest [IRremoteESP8266 v2.7.4](https://github.com/crankyoldgit/IRremoteESP8266/releases/tag/v2.7.4) release.
